explorer/src/components/TableToolbar.tsx

```diff
@@ -30,23 +30,32 @@ export const TableToolbar: React.FC<TableToolBarProps> = ({
         }}
       >
         <Select
-          labelId="demo-simple-select-label"
-          id="demo-simple-select"
+          labelId="simple-select-label"
+          id="simple-select"
```

Comment:
OOC, Any reason this isn't a data-testid?

Reply:
This was already baked into the framework from the looks of things, I just dropped the `demo` from the naming, because it isn't a demo anymore 😉